src/index.js

```diff
@@ -122,7 +121,7 @@ module.exports = exports = options => {
           send(rooms.get(key), clients, data, options))
       })
     } else {
-      send(socks.keys(), clients, data, options)
+      send(new Set([...socks.keys()]), clients, data, options)
```

Comment:
Using `--harmony` is not recommended for production environment and I wouldn't expect users to use the flag anyway :)